editor/src/core/shared/element-template.ts

```diff
@@ -1607,6 +1607,7 @@ export interface SpecialSizeMeasurements {
   float: string
   hasPositionOffset: boolean
   textDirection: string
```

Comment:
Instead of `hasTransform`, I would add a `transform: 'translate' | 'scale' | 'rotate'`... etc
It is just as easy to extract this information, and maybe in the future we need more than a boolean.

Reply:
That would be useful, unfortunately the computed value is a matrix form which is not handled by our css parser yet. It could be a nice improvement for later!